Parsedown.php

```diff
@@ -75,6 +75,15 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
```

Comment:
You'd want to declare this property as a `protected` one and not just leave it to be declared in runtime.